app/state.go

```diff
@@ -49,6 +49,9 @@ func (state *State) List() (ret []string, err error) {
 	}
 	ret = make([]string, 0, len(dirContent))
 	for _, info := range dirContent {
+		if info.Name() == "sap_force_latency" {
```

Comment:
Can you explain why this was added? It seems a bit surprising to me that the List() method should do this filtering already... maybe it would make sense to have a separate filter method which takes the output of List() as its input, and filters out `sap_force_latency`, and which can then also have a comment explaining why this key is filtered?

Reply:
The force_latency handling is a totally new part of saptune. And it's not really finished. I'm still in discussion with Sören how exactly we want to present the related values to the customer and how to handle it internal.
So the old method using the tuned configuration file for force_latency gives the customer the parameter-value pair 'force_latency=70'. tuned uses this value to set the latency by /dev/cpu_dma_latency file. This implies a daemon running, because the file /dev/cpu_dma_latency needs to be hold open all the time the latency value should be valid.
For saptune Sören found another solution based on /sys/devices/system/cpu/cpu\* files. That's highly welcome, because working with a daemon is possible but not really preferred.
And to not confusing our customers to much, I want to prevent the  'force_latency=\<value\>' pair for choosing a latency value. 
But now I have the problem, where to get the value of 'force_latency' from. For the beginning I can read the value from /dev/cpu_dma_latency file. And this can be the 'old' value for a later revert. But where to remember the 'current' active latency value to display during a 'verify' or 'simulate'? As I do not write to /dev/cpu_dma_latency any longer, I can not retrieve the current value from there. And the files under /sys/devices/system/cpu/cpu\* do not cover this value (they work on another way - see https://bugzilla.suse.com/show_bug.cgi?id=1109584 for a very good explanation)
So for the first step I misused the already available location /var/lib/saptune/saved_state to store the 'current' value of force_latency. It was the simplest way for me. But as it is NOT a stored note state I need to exclude it from the list of stored note states.
The discussion with Sören about the presentation of force latency to the customer is at the moment ongoing. So it may be possible that we decide to change the 'force_latency=\<value\>' pair to anything else and then I do not need this file any longer. But if we do not change this behavior, then I will move this file to another location and remove this lines from List()

Reply:
Hm, I see. Thanks for the explanation :) Maybe you could add a comment noting that this is related to an ongoing discussion and subject to change?